saleor/graphql/checkout/mutations.py

```diff
@@ -989,13 +989,27 @@ class Meta:
         error_type_class = CheckoutError
         error_type_field = "checkout_errors"
 
+    @staticmethod
+    def clean_email(email):
```

Comment:
Should we add similar validation in checkout complete? Maybe It's planed for separate PR.

Reply:
It will be done in separate PR :)